### PR TITLE
Formula default value

### PR DESCRIPTION
**Pull request by Fabi755**
The missing commits for issue #16 contain
- Add the `fullCalcOnLoad` value
- Allow default values on formulas

I think we should add a description for the topic default-value anywhere, but I'm not sure which place should be good for this and how describe the problematic simple.

Followed calls are now valid:
`Sheet.set_cell("XY", {:formula, "SUM(1,2,3)"})`
`Sheet.set_cell("XY", {:formula, "SUM(1,2,3)", value: "6"})`

Tested behavior with default value:
Microsoft Excel: Calcluate
Libre Office: **NOT** Calculate
Numbers: Calculate

Tested on:
Microsoft Excel for Mac 15.31
Libre Office 5.2.5.1 on Mac
Numbers 4.0.5 on Mac

### CODE REVIEW

File: lib/elixlsx/xml_templates.ex [hunk 1]
```diff
@@ -195,10 +198,15 @@ defmodule Elixlsx.XMLTemplates do
 
           case content_type do
             :formula ->
+              if not is_nil(content_opts[:value]) do
+                value = "<v>#{content_opts[:value]}</v>"
```
Comment (xou) [inline lib/elixlsx/xml_templates.ex]: value should have a default I believe (as in `value = if ... :do ... :else ""` etc)

**Comment by unti1x** [comment]
Actually, testing of private methods is a bad practice, because sooner or later you'll start testing of realization, not behavior. Fortunately, you can't do that easy in Elixir.

**Comment by Fabi755** [comment]
> I feel like xl_sheet_cols is getting quite large and probably requires better testing at this point. You can't unfortunately test defp's that easy, so it'll require some restructuring... Probably best at a later time.

You are right, some parts (xl_sheet_cols in example) should be better structured and more tests are not wrong. But we should discuss this topic in an extra issue.

**Comment by xou** [comment]
Cheers, thank you!

**Pull request merged**